biometric-login/test/configs/wdio.ios.sauce.sim.conf.js

```diff
@@ -87,7 +154,38 @@ config.capabilities = [
         locale: 'en',
         // Group builds by build name
         build: buildName,
+        name: 'iOS 13.4 iPad FaceID App',
     },
+
+    // iOS iPad 14.0 FaceID
+    {
+        // The defaults you need to have in your config
+        deviceName: 'iPad Simulator',
+        platformName: 'iOS',
+        platformVersion: '14.0',
+        orientation: 'PORTRAIT',
+        // The path to the app
+        app: 'storage:filename=sample-app-ios.zip',
+        // Read the reset strategies very well, they differ per platform, see
+        // http://appium.io/docs/en/writing-running-appium/other/reset-strategies/
+        noReset: true,
+        newCommandTimeout: 240,
+        // Always default the language to a language you prefer so you know the app language is always as expected
+        language: 'en',
+        locale: 'en',
+        // Group builds by build name
+        build: buildName,
+        name: 'iOS 14.0 iPad FaceID App',
+    },
+
 ];
 
+const appiumVersion = process.env.npm_config_appium_version
```

Comment:
Can you put the constant at the top of the file, it's more of a preferred way instead of a mandatory one

Reply:
moved into a shared conf